test/unit/content_workflow_presenter_test.rb

```diff
@@ -0,0 +1,71 @@
+# encoding: utf-8
+require 'test_helper'
+
+class ContentWorkflowPresenterTest < ActiveSupport::TestCase
+  should "provide a CSV export of content workflow" do
+    action_1 = Action.new(
+      request_type: "create",
+      created_at: "2016-01-07 17:41:57"
+    )
+
+    action_2 = Action.new(
+      request_type: "request_review",
+      created_at: "2016-01-11 10:21:00"
+    )
+
+    action_3 = Action.new(
+      request_type: "send_fact_check",
+      created_at: "2016-01-17 12:11:33"
+    )
+
+    action_4 = Action.new(
+      request_type: "request_amendments",
+      created_at: "2016-01-20 17:41:28"
+    )
+
+    action_5 = Action.new(
+      request_type: "send_fact_check",
+      created_at: "2016-03-07 17:31:44"
+    )
+
+    transaction_edition = TransactionEdition.new(
+      title: "Register to vote (armed forces)",
+      slug: "register-to-vote-armed-forces",
+      state: "published",
+      assignee: "Ray Khan",
+      actions: [action_1, action_2, action_3, action_4, action_5]
+    )
+
+    guide_edition = GuideEdition.new(
+      title: "The Queen's Awards for Enterprise",
+      slug: "queens-awards-for-enterprise",
+      state: "published",
+      assignee: "Constance Cerf",
+      actions: [action_1, action_2, action_3, action_4, action_5]
+    )
+
+    Edition.stubs(:published).returns([transaction_edition, guide_edition])
```

Comment:
I'm not sure any modification to the `Edition` model is needed, the scope as I understand it finds published editions and orders them, this can be achieved by chaining existing scopes and order clauses as mentioned above. This could be done here: https://github.com/alphagov/publisher/pull/462/files/78f0ea9a7805855f91c30245973be7598a34bb88#diff-ef36a8aacde58db76a4c63c23399c589R36
with
```ruby
Edition.published.order(created_at: :desc)
```
couldn't it?